clusters/data_node_type.go

```diff
@@ -89,6 +94,19 @@ type NodeType struct {
 	Graviton              bool                          `json:"is_graviton,omitempty"`
 }
 
+func (nt NodeType) shouldBeSkipped() bool {
+	shouldBeSkipped := false
+	if nt.NodeInfo != nil {
+		for _, st := range nt.NodeInfo.Status {
+			if st == CloudProviderNodeStatusNotAvailableInRegion || st == CloudProviderNodeStatusNotEnabled {
+				shouldBeSkipped = true
```

Comment:
why not immediately return here? would be more robust.

Reply:
addressed all comments